### DISCUSSION TITLE
possible to offset $this:column

### DISCUSSION THREAD

**Discussion by SmutjeHannes** [bitfocus/companion#2978] [Q&A] [answered]
Is it possible to offset $this:column value?
Because it starts with 0 but I would need it to be 1. 
Want to display track names etc. from reaper and have track 1 display its name on button 1 and track 2 on button 2 and so on.
Currently I am using parseVariables('$(internal:custom_track_name$(this:column))') this is what I want to change. Because on Button 1 I want to call custom_track_name1 and not custom_track_name0.
Any ideas how to achieve this?

**Comment by Julusian** [comment, marked as answer]
I think this should work:
```
parseVariables(`$(internal:custom_track_name${ $(this:column) + 1 })`)
```

Using string interpolation to generate the variable name

**Reply by SmutjeHannes** [reply]
It should...but it isn't!

**Reply by dnmeid** [reply]
This doesn't work in the text context with variables substitution, it is working in expressions.

**Reply by SmutjeHannes** [reply]
Thanks a LOT  @dnmeid !!!
